scripts/check-sidebar-lazy-layout.py

```
@@ -1,741 +1,283 @@
-            if re.search(r"\b" + re.escape(name) + r"\b", body):
-                violations.append(
-                    "{0}(...) applies the custom Layout `{1}` to the sidebar rows. "
-                    "A custom Layout wrapping the LazyVStack measures it on every "
-                    "pass (the #6210 force-measure shape, regardless of the type's "
-                    "name); size the rows with .frame(minHeight:) instead.".format(
-                        func_name, name
+            for pattern in LAYOUT_MUTATION_PATTERNS:
+                if pattern.search(body):
+                    violations.append(
+                        f"{filename}.{callback} mutates/reconfigures the table from a layout callback"
                     )
```

Ruff (0.15.21)

[warning] Use `list.extend` to create a transformed list

(PERF401)